### PR TITLE
add third reality T&H and smart switch ota file

### PR DESCRIPTION
**Pull request by weihuan1111**
@Koenkk please help to review the pull request, thanks.

### CODE REVIEW

**Comment by Koenkk** [comment]
For which device is this firmware? (I need the model number)

**Comment by weihuan1111** [comment]
@Koenkk The devices is temperature & humidity sensor, and the model is 3RTHS24BZ.

File: index.json [hunk 1]
```diff
@@ -2067,5 +2075,13 @@
         "sha512": "91605b5a7f965ac34a1e72bd2e2adba7abea877182b043b6d21e78ef1aa68005eba7011c548116ef2df74d29c7036e6a2616c65597705fd64d01d2f528ceb986",
         "url": "https://github.com/Koenkk/zigbee-OTA/raw/master/images/Innr/1166-0313-31016610-upgradeMe.ota",
         "path": "images/Innr/1166-0313-31016610-upgradeMe.ota"
+    },
```
Comment (weihuan1111) [inline index.json:2078]: @Koenkk This device is Smart Switch, the model is 3RSS009Z

File: index.json [hunk 2]
```diff
@@ -2041,6 +2041,14 @@
         "url": "https://github.com/Koenkk/zigbee-OTA/raw/master/images/Ledvance/Zigbee3toDALI_1008655B.ota",
         "path": "images/Ledvance/Zigbee3toDALI_1008655B.ota"
     },
+    {
+        "fileVersion": 33,
+        "fileSize": 124946,
+        "manufacturerCode": 4659,
+        "imageType": 54185,
+        "sha512": "a696fbeb2fc2f899d23f605099add1930b9a6c5ee69f07fb04e6d22ce8349c676898fcaab1c3cb00981ff37528a61116905ca36ffdaea4f6425d47af3cb7790a",
+        "url": "https://tr-zha.s3.amazonaws.com/firmwares/TRTL_ThermalSensor_PROD_OTA_V21_1.00.21.ota"
```
Comment (weihuan1111) [inline index.json:2050]: @Koenkk This device is temperature $ humidity sensor, the model is 3RTHS24BZ

File: index.json [hunk 3]
```diff
@@ -2067,5 +2075,13 @@
...
         "url": "https://github.com/Koenkk/zigbee-OTA/raw/master/images/Innr/1166-0313-31016610-upgradeMe.ota",
         "path": "images/Innr/1166-0313-31016610-upgradeMe.ota"
+    },
+    {
+        "fileVersion": 18,
+        "fileSize": 146322,
+        "manufacturerCode": 4659,
+        "imageType": 54181,
+        "sha512": "0864730bdf1d4455681fd4866ec934f325114a50c2375985dffd36bbc4d7ea9d8464f1bcfa3241482a3c4afedbfc32c1fbdec6e5a3f4629b765393768a198889",
+        "url": "https://tr-zha.s3.amazonaws.com/firmwares/SmartSwitchGen3_Zigbee_PROD_V12_v1.00.12.ota"
```
Comment (Koenkk) [inline index.json:2085]: And for which device is this OTA?

**Comment by weihuan1111** [comment]
@Koenkk How can I verify OTA locally？

**Comment by Koenkk** [comment]
See https://www.zigbee2mqtt.io/guide/configuration/ota-device-updates.html#ota-index-override-file

**Pull request closed without merging**